csfieldguide/chapters/content/en/complexity-and-tractability/sections/the-travelling-salesman-problem.md

```diff
@@ -100,15 +100,15 @@ Just like in that activity you listed all the possible ordering for the values i
 How many possible routes are there for the larger example you have generated?
 How is this related to permutation sort, and factorials?
 How long would it take to calculate the shortest route in your map, assuming the computer can check 1 billion (1,000,000,000) possible routes per second?
-(i.e. it can check one route per nanosecond) What can you conclude about the cost of this algorithm?
+(i.e. it can check one route per nanosecond). What can you conclude about the cost of this algorithm?
 Would this be a good way for the cray fisher to decide which path to take?
```

Comment:
Do you have a suggestion for a replacement? I considered fisherman but not sure what our stance is on using words with '-man' in them. I googled it and apparently people tend to use 'fisher' now but it sounds strange to me.

Reply:
Our stance, according to the docs, is to use gender neutral terms when referring to a hypothetical person, but that is very ambiguous. Since: people who catch crayfish don't seem to have their own term; 'man' itself (even before you add other characters) is historically gender neutral; and noone's going to make too much of a fuss over a basic example, I think it's okay to say fisherman

Reply:
We had an office discussion and a quick Google - apparently crayfisher is the right term so will go with that.

Reply:
Fine with me